be/src/olap/storage_policy.h

```diff
@@ -62,20 +62,35 @@ std::vector<std::pair<int64_t, int64_t>> get_storage_policy_ids();
 
 struct StorageResource {
```

Comment:
`LocalFileSystem` is also a kind of `StorageResource`, we may take it into consideration in the future.